Libraries/Components/StatusBar/StatusBar.js

```diff
@@ -135,6 +156,13 @@ const StatusBar = React.createClass({
 
     // Provide an imperative API as static functions of the component.
     // See the corresponding prop for more detail.
+
+    /**
+     * Show or hide the status bar
+     * @param {boolean} hidden The dialog's title.
+     * @param {StatusBarAnimation=} animation Optional animation when
```

Comment:
I guess for this case we want to be a bit more specific and add some comments to the aprams